presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestGrantRevoke.java

```diff
@@ -121,6 +125,53 @@ public void testPublic()
         assertThat(aliceExecutor.executeQuery(format("SELECT * FROM %s", tableName))).hasNoRows();
     }
 
+    @Test(groups = {AUTHORIZATION, HIVE_CONNECTOR, PROFILE_SPECIFIC_TESTS})
+    public void testTableOwnerPrivileges()
+    {
+        String tableName = "alice_table";
```

Comment:
The reason of not having a DROP TABLE at the end is that we want to keep the generated tables since that will help in debugging if there is a test failure. Don't we follow this philosophy everywhere in our product tests? For ex. in `shouldCreateTableAsSelect()`, we don't drop the table either.